app/assets/javascripts/components/tickets/tickets_table_row.jsx

```diff
@@ -7,10 +7,13 @@ import TicketOwnerHandler from './ticket_owner_handler';
 const TicketsTableRow = ({ ticket }) => {
   return (
     <tr className={ticket.status === 0 ? 'table-row--faded' : ''}>
+      <td className="w10">
+        {ticket.sender.real_name || ticket.sender.username || ticket.sender_email || 'Unknown User Record' }
```

Comment:
I wonder if we should extract this out, at this point. We could define a variable above and still leave the default below.
```js
const { sender, sender_email } = ticket;
const user = sender.real_name || sender.username || sender_email;
// ...
{ user || 'Unknown User Record' }
```